packages/frontend/admin/src/modules/ai/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Separator } from '@affine/admin/components/ui/separator';
+import { cn } from '@affine/admin/utils';
+import * as ScrollAreaPrimitive from '@radix-ui/react-scroll-area';
+
+import { Layout } from '../layout';
+import { Prompts } from './prompts';
+
+export function Ai() {
+  return <Layout content={<AiPage />} />;
```

Comment:
do not show ai page if serverConfig.features not including ai

Reply:
directly hide ai config in admin for now.
this is implementation is not even usable, and @darkskygit will help it